src/Whoops/Resources/js/whoops.base.js

```diff
@@ -94,20 +104,34 @@ Zepto(function($) {
     return actionMsg;
   }
 
+  function scrollIntoView($node, $parent) {
```

Comment:
The old code always keeps the active frame at the top. Check the blue line in the active frame, the first time it makes a jump, afterwards it stays on top
https://www.dropbox.com/s/s916mej3hva6fk6/before.mov?dl=0

The new code does what you would expect to do
https://www.dropbox.com/s/tjh95qj52xkwd3u/after.mov?dl=0